src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1995,4 +2003,15 @@ public int getLastContactInHours()
     {
         return lastContactInHours;
     }
+
+    @Override
+    public boolean isCanBeAutoDeleted()
+    {
+        return canBeAutoDeleted;
+    }
```

Comment:
there is an @override ? in mine anyway.... i'm pushing my latest in a few minute hopefully that'll show the @override